src/js/views/hallOfFame.js

```diff
@@ -30,11 +30,19 @@ async function updatePlayers(inputs, updateEvents) {
                 min: 0,
                 per: 0,
             };
+            players[i].teamSums = {};
             for (let j = 0; j < players[i].stats.length; j++) {
+                const team = players[i].stats[j].tid;
                 if (players[i].stats[j].gp * players[i].stats[j].min * players[i].stats[j].per > players[i].bestStats.gp * players[i].bestStats.min * players[i].bestStats.per) {
                     players[i].bestStats = players[i].stats[j];
                 }
+                if (players[i].teamSums.hasOwnProperty(team)) {
+                    players[i].teamSums[team] += players[i].stats[j].gp * players[i].stats[j].min * players[i].stats[j].per;
```

Comment:
Could you quickly explain the 67 and 30 in the denominator there? What are they referencing?

Also, I'm assuming `players[i].stats.min` in that equation is total season minutes, which would therefore be the same as mpg \* GP. If that's the case, then I guess 67 & 30 are just constant factors anyway.

I'm deferring to your judgement here, but take a look at this HoF example statline: http://imgur.com/yWffuSn

I've sorted it by EWA, which shows the 2021 NYC season as his top season. However, if you actually perform the GP \* MPG \* PER calculation, you see the following:

* 2021 NYC: 82 \* 30 \* 21.7 = 53,382 (10.4 EWA)
* 2028 TPA: 82 \* 31.3 \* 20.7 = 53,128.6 (9.9 EWA)
* 2019 MIN: 82 \* 34 \* 19.9 = 55,481.2 (9.8 EWA)

By PER, the 2019 MIN season is the best, despite having lower EWA than the 2021 NYC season. I'm really curious why this is the case - I actually assumed that your EWA equation actually did something funky to include a factor of actual games won and thought this accounted for some of the variation I'm seeing. If no such factor exists, why is the relationship between EWA and PER not perfectly linear?